app/importer/client/admin/adminImportHistory.html

```diff
@@ -17,7 +17,7 @@
 								<h1>{{ statusMessage}}</h1>
 
 								<span style="font-weight: 600;">{{_ "Import_Type"}}:</span>
-								<span>{{ type}} [{{ importerKey}}]</span>
+								<span>{{translate type}} [{{ importerKey}}]</span>
```

Comment:
change to 
`<span>{{_ type}} [{{ importerKey}}]</span>`
the "_" symbol is a global helper that does the same thing as the one you created.